simprod/simjob/simjob.py

```diff
@@ -640,6 +640,10 @@ def prepare( self, **kwargs ):
 					self._options["logdestdir"]  = "{0}/{1}".format( self.options["logdir"], self.options["subdir"])
 				
 			self._destdir = "{0}/{1}/{2}/{3}".format( self.__destination(), self._evttype, self._year, self._simcond)
+			
+			if self._redecay: 
+				self._proddir += "ReDecay"
```

Comment:
Just a small thing but maybe you could add an underscore

self._proddir += "_ReDecay"